README.md

```diff
@@ -70,22 +70,167 @@ source for the project.  Packaging and cataloging are covered below.
 
 # Using The BrainIO Python Tools
 
-There are three main ways to use the Python BrainIO tools:
-* To access and analyze stimuli and data that are already packaged and cataloged.
+## Basic API
+
+The most commonly used methods of the BrainIO API are:  
+
+* **List Catalogs**: `brainio.list_catalogs()` lists the identifiers of all the BrainIO catalogs installed in your current Python environment.  
+* **List Stimulus Sets**:  `brainio.list_stimulus_sets()` lists the identifiers of all the BrainIO stimulus sets available via your installed catalogs.  
+* **List Data Assemblies**:  `brainio.list_assemblies()` lists the identifiers of all the BrainIO data assemblies available via your installed catalogs.  
+
+
+* **Stimulus Set From Files**:  `brainio.stimuli.StimulusSet.from_files(csv_path, dir_path)` loads into memory a stimulus set contained in the provided files.  
+* **Data Assembly From Files**:  `brainio.assemblies.DataAssembly.from_files(file_path, group=None, stimulus_set_identifier=None, stimulus_set=None, **kwargs)` loads into memory a data assembly contained in the provided file.  Usually called from a subclass of DataAssembly.  
```

Comment:
I would not show the additional default parameters `group=None, stimulus_set_identifier=None, stimulus_set=None, **kwargs` here